internal_cert_management/scripts/update_endpoints.sh

```
@@ -0,0 +1,81 @@
+    echo "⚠️  Certificate validation is DISABLED (--force)"
+fi
+
+cd "$ANSIBLE_PATH"
+
+# Build extra vars for ansible
+EXTRA_VARS=""
+if [ "$FORCE_SKIP_VALIDATION" = true ]; then
+    EXTRA_VARS="-e force_skip_validation=false"
+fi
+
+for subdomain in ${subdomains[@]}
```

Shellcheck (0.11.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)